server.js

```diff
@@ -0,0 +1,24 @@
+require('./src/config');
+const express = require('express');
+const argv = require('minimist');
+const path = require('path');
+const healthcheck = require('express-healthcheck');
+
+const args = argv(process.argv.slice(2));
+const HTTP_PORT = process.env.PORT || args.port || 3000;
```

Comment:
Let's not allow passing args in like this. We have an `env` system in place, let's just use that.  If people want to override it on the command line, they can with `PORT=12345 npm start`.